src/views/VallisFish.vue

```diff
@@ -63,29 +63,42 @@
             invert="true"
           />
         </template>
-        <template slot="time" slot-scope="data">
-          <i class="fas fa-times-circle"></i>
+        <template slot="time.string" slot-scope="data">
+          <span v-bind:id="data.item.name + '-time'">
+            <i v-if="data.item.time.cold.appear" class="fas fa-lg fa-snowflake" style="color:#32d2ff" />
+            <i v-if="data.item.time.cold.prefer" class="fas fa-lg fa-arrow-left mx-1"></i>
+            <i v-if="data.item.time.warm.prefer" class="fas fa-lg fa-arrow-right mx-1"></i>
+            <i v-if="data.item.time.warm.appear" class="fas fa-lg fa-sun" style="color:#f9a805" />
+          </span>
+          <b-tooltip v-bind:target="data.item.name + '-time'" :title="data.item.time.string" />
         </template>
         <template slot="rarity" slot-scope="data">
           <FishImg
-            v-if="data.item.rarity === '1'"
+            v-if="data.item.rarity === '1-Common'"
```

Comment:
still not loving this... could we do....
```html
<FishImg
    type="common"
    :item="data.item.rarity.splice(2).toLowerCase()"
    :name="data.item.rarity.splice(2)"
    :title="data.item.rarity.splice(2)"
    width="20"
  />
````

I think this'd shorten it up and make it a lot simpler